fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/SqlToScalarOperatorTranslator.java

```diff
@@ -490,4 +499,17 @@ public ScalarOperator visitCloneExpr(CloneExpr node, Void context) {
             return new CloneOperator(visit(node.getChild(0)));
         }
     }
+
+    static class IgnoreSlotVisitor extends Visitor {
+        public IgnoreSlotVisitor() {
+            super(new ExpressionMapping(new Scope(RelationId.anonymous(), new RelationFields())),
+                    Collections.EMPTY_LIST);
+        }
+
+        @Override
+        public ScalarOperator visitSlot(SlotRef node, Void context) {
+            return new ColumnRefOperator(node.getSlotId().asInt(),
+                    node.getType(), node.getColumnName(), node.isNullable());
```

Comment:
IgnoreSlotVisitor seems not handling slot, there is a little mismatch

Reply:
IgnoreSlotVisitor ignore the slot, which is consistent with the name